core/node/node_framework/src/implementations/layers/vm_runner/playground.rs

```diff
@@ -79,10 +82,15 @@ impl WiringLayer for VmPlaygroundLayer {
             window_size: self.config.window_size,
             reset_state: self.config.reset,
         };
+        let storage = if let Some(path) = self.config.db_path {
+            VmPlaygroundStorageOptions::Rocksdb(path)
+        } else {
+            VmPlaygroundStorageOptions::Snapshots { shadow: false }
```

Comment:
I haven't added an option to configure storage shadowing for snapshots. May do so separately.